main/Jacek_Kaczmarski/Lekcja_historii_klasycznej.tex

```diff
@@ -0,0 +1,41 @@
+\tytul{Lekcja historii klasycznej}{sł. muz. Jacek Kaczmarski}{Jacek Kaczmarski}
+\begin{text}
+	\vin Gallia est omnis divisa in partes tres\\
+	\vin Quarum unam incolunt Belgae aliam Aquitani\\
+	\vin Tertiam qui ipsorum lingua Celtae nostra\\
+	\vin Galli appellantur\\
+	\vin Ave Caesar morituri te salutant!
+
+	Nad Europą twardy krok legionów grzmi\\
+	Nieunikniony wróży koniec republiki\\
+	Gniją wzgórza galijskie w pomieszanej krwi\\
+	A Juliusz Cezar pisze swoje pamiętniki
+
+	\vin Gallia est omnis...
+
+	Pozwól, Cezarze, gdy zdobędziemy cały świat\\
+	Gwałcić, rabować, sycić wszelkie pożądania\\
+	Proste prośby żołnierzy te same są od lat\\
+	A Juliusz Cezar milcząc zabaw nie zabrania
+
+	\vin Gallia est omnis...
+
+	Cywilizuje podbite narody nowy ład\\
+	Rosną krzyże przy drogach od Renu do Nilu\\
+	Skargą, krzykiem i płaczem rozbrzmiewa cały świat\\
+	A Juliusz Cezar ćwiczy lapidarność stylu!
+
+	\vin Gallia est omnis...
+\end{text}
+\begin{chord}
+    C G\\
+    d E\\
+    a\\
+    F\\
+    F C G C
+    
+    C G\\
+    d E\\
+    a F\\
+    F C G C
```

Comment:
Dobrze by było dodać tutaj kolejne powtórzenia chwytów, sposób na to już znasz :)

Reply:
Zrobione